compressor/Server/MainHandler.py

```diff
@@ -66,21 +69,32 @@ def record_and_compress():
             decibels = 20 * math.log10(audio_levels)
 
             # dB 0 < x < 100 -- Normal & Acceptable use
-            not_clipping = (decibels >= 0 and decibels  < 90)
+            not_clipping = (decibels >= 0 and decibels + 50 < 100)
             if not_clipping:
                 stream.write(data, chunk)
 
             # x >= 100
             else:
+                chunk_temp = chunk
 
                 # do your bidding sir
-                sound = AudioSegment(data, sample_width=sample_width, channels=channels, frame_rate=sample_rate)
```

Comment:
I'm approving your PR but I changed a few things back so that the Pi isn't going to be creating thousands of temp files.